services/sidecar/src/simcore_service_sidecar/mpi_lock.py

```diff
@@ -125,7 +133,14 @@ def acquire_mpi_lock(cpu_count: int) -> bool:
     Will try to acquire a distributed shared lock.
     This operation will last up to 2 x config.REDLOCK_REFRESH_INTERVAL_SECONDS
     """
-    from .utils import wrap_async_call
+    reply_queue = multiprocessing.Queue()
+    multiprocessing.Process(
+        target=process_worker, args=(reply_queue, cpu_count), daemon=True
+    ).start()
+
+    response = reply_queue.get()
+    if not isinstance(response, bool):
```

Comment:
In what case would you get a non boolean?

Reply:
you are correct, no need for this.